packages/metro-transform-worker/src/__tests__/index-test.js

```diff
@@ -211,6 +211,36 @@ it('transforms an es module with regenerator', async () => {
   ]);
 });
 
+it('transforms async generators', async () => {
+  const result = await Transformer.transform(
+    baseConfig,
+    '/root',
+    'local/file.js',
+    'export async function* test() { yield "ok"; }',
+    {
+      dev: true,
+      type: 'module',
+      unstable_transformProfile: 'hermes-stable',
```

Comment:
Can we remove this line and test the default transform profile instead? Alternatively we can have a separate test for `hermes-stable`, but the main test should exercise the default profile.

Reply:
Sure, will do